sdk/eventhubs/azure-messaging-eventhubs/perf-tests.yml

```diff
@@ -0,0 +1,31 @@
+Service: storage-blob
+
+Project: sdk/storage/azure-storage-perf
+
+PrimaryPackage: 'com.azure:azure-messaging-eventhubs'
+
+PackageVersions:
+- 'com.azure:azure-messaging-eventhubs': 5.15.5
+  'com.azure:azure-core': 1.39.0
+  'com.azure:azure-core-amqp': 2.8.5
+  'com.azure:azure-core-http-netty': 1.13.3
+  'com.azure:azure-core-http-okhttp': 1.11.9
+  'io.projectreactor:reactor-core': 3.4.27
+- 'com.azure:azure-messaging-eventhubs': source
+  'com.azure:azure-core': source
+  'com.azure:azure-core-amqp': source
+  'com.azure:azure-core-http-netty': source
+  'com.azure:azure-core-http-okhttp': source
+  'io.projectreactor:reactor-core': source
+
+Tests:
+- Test: send-event
```

Comment:
Maybe make this test name more descriptive and consistent with Python: `send-event-batch`?